R/util.R

```diff
@@ -30,12 +30,22 @@ SetApiKey = function(key) {
   Sys.setenv(API_KEY = key)
```

Comment:
ah, we're actually setting it in the env...

do we rely on it from the environment, vs what's stored in the dc library (is this solely for the way query is implemented?)

Reply:
Yes, the environment variable would just be for the query function

Reply:
Actually it is also useful for not losing the api key for testing.

Reply:
we can remove this from the general library, and just have testers set an API key to environment before running tests